### ISSUE TITLE
Failed to create a Python Dev container

### ISSUE THREAD

**Issue by dbaeumer** [microsoft/vscode-remote-release#9701]
Steps to Reproduce:

1.  New VS Code window
2.  New Dev Container
3. Pick Python

![Image](https://github.com/microsoft/vscode/assets/1931590/bd4ac9f0-bc9f-4dfa-af3a-0fb5a99c534c)

**Comment by dbaeumer** [comment]
here a log when trying to clone a repository into a dev container:



[log.txt](https://github.com/microsoft/vscode-remote-release/files/14759093/log.txt)

**Comment by dbaeumer** [comment]
And here the log for creating a Python Dev container


[log.txt](https://github.com/microsoft/vscode-remote-release/files/14759116/log.txt)

**Comment by chrmarti** [comment]
It can't mount the Wayland socket:
```
docker: Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /mnt/wslg/runtime-dir/wayland-0.
```

Is this with a Docker Desktop install?

You can disable this feature in the user settings:
![Image](https://github.com/microsoft/vscode-remote-release/assets/9205389/75e2f880-3778-4a37-a1c0-02a7005d87b7)

**Comment by dbaeumer** [comment]
Yes, this is with a docker desktop install

**Comment by dbaeumer** [comment]
Reopening since it is IMO not fixed. @chrmarti please close if you think otherwise.

**Comment by chrmarti** [comment]
Fixed with #9293
